src/_includes/banner.html

```diff
@@ -5,7 +5,7 @@
     https://github.com/flutter/website/issues/3897
   {% endcomment %}
   <div class="site-banner site-banner--default" role="alert">
-    Flutter 2.2 is here. <a href="https://medium.com/flutter/announcing-flutter-2-2-at-google-i-o-2021-92f0fcbd7ef9">Learn more</a>. Also, see
-   <a href="/docs/whats-new#may-18-2021-google-io-edition">What's new on this site</a>, which includes a list of the new instructor-led video workshops!
+    <a href="https://medium.com/flutter/announcing-flutter-2-2-at-google-i-o-2021-92f0fcbd7ef9">Get started with Flutter 2.2.</a>. See
```

Comment:
There are two dots in a row.

```suggestion
    <a href="https://medium.com/flutter/announcing-flutter-2-2-at-google-i-o-2021-92f0fcbd7ef9">Get started with Flutter 2.2.</a> See
```